docs/concept-guide/scaling-fireproof.md

```diff
@@ -4,31 +4,31 @@ sidebar_position: 5
 
-There’s no limit to the number of databases a Fireproof install can have, so naming them with UUIDs or other random identifiers makes sense. Databases can cheaply fork, so you can share the same initial storage across your user base, and write their forks as individual diffs, merge and publish a new canonical main branch, and repeat the process while minimizing duplication. See the storage architecture section for more detail on the encrypted block store.
+There’s no limit to the number of ledgers a Fireproof install can have, so naming them with UUIDs or other random identifiers makes sense. Ledgers can cheaply fork, so you can share the same initial storage across your user base, and write their forks as individual diffs, merge and publish a new canonical main branch, and repeat the process while minimizing duplication. See the storage architecture section for more detail on the encrypted block store.
 
 ## Compaction
 
-Each database transaction is written to a new file or key as a lightweight encrypted diff, including links to the previous diff to allow reconstructing the complete dataset. After a certain number of transactions have accumulated, the set of diffs to load creates its own performance challenges. To address this, Fireproof periodically compacts the dataset to a single file, by default when the number of diffs is greater than 100.
+Each ledger transaction is written to a new file or key as a lightweight encrypted diff, including links to the previous diff to allow reconstructing the complete dataset. After a certain number of transactions have accumulated, the set of diffs to load creates its own performance challenges. To address this, Fireproof periodically compacts the dataset to a single file, by default when the number of diffs is greater than 100.
 
-There are numerous compaction optimizations that can be implemented. Before 1.0, we need to decide on semantics for data preservation. Most databases allow for a system purge to remove old versions of data, either on a key or historic basis. Currently, Fireproof compaction preserves all database history, but discards unreferenced internal index blocks, etc. Each compaction links to the dataset it replaces, so it’s possible to reconstruct history from immutable archives even from a compacted lean dataset. This is a great topic to bring up in the [Discord](https://discord.gg/xC55bhgf6n).
+There are numerous compaction optimizations that can be implemented. Before 1.0, we need to decide on semantics for data preservation. Most ledgers allow for a system purge to remove old versions of data, either on a key or historic basis. Currently, Fireproof compaction preserves all ledger history, but discards unreferenced internal index blocks, etc. Each compaction links to the dataset it replaces, so it’s possible to reconstruct history from immutable archives even from a compacted lean dataset. This is a great topic to bring up in the [Discord](https://discord.gg/xC55bhgf6n).
```

Comment:
first databases can stay

Reply:
the only fix that is blocking merge is this

Reply:
addressed in 0d1834